### PR TITLE
Review initial code

### PR DESCRIPTION
**Pull request by C-D-Lewis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# health-steps-remote
+
+Basic app that can send steps data to a remote server. Default behavior is to
+upload the last 15 minutes worth of minute data after that interval, but can be
+configured in `src/config.h`.
+
+The server receiving the data is selected with the `NODE_SERVER_URL` variable in
+`src/js/pebble-js- app.js`, and in this example is a simple Node server with a
```
Comment (orviwan) [inline README.md]: extra space in pebble-js-app.js

Comment (orviwan) [inline README.md]: is a simple Node server.

Revise next sentence, it doesn't read very clearly.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+function (`onNewData()`) broken out to allow use of the received data array as
+preferred.
```
Comment (orviwan) [inline README.md]: Would be worth just explaining how to setup, configure and launch the server?
1. Launch the server:
   $ node app.js
2. Edit the NODE_SERVER_URL in pebble-js-app.js to point to your computer IP.
3. Build the Pebble application and install the PBW file on your Pebble

Please note: your mobile phone must have network access to the computer running the server.

File: src/config.h [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#include <pebble.h>
+
+#define DATA_NUM_ENTRIES 15
```
Comment (orviwan) [inline src/config.h]: Can you put some comments for each setting?

File: src/js/pebble-js-app.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
+var VERSION = "1.0";
+var DEBUG = true;
+var NUM_ITEMS = 15;
```
Comment (orviwan) [inline src/js/pebble-js-app.js]: Should this value come from the C side?

File: README.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+# health-steps-remote
```
Comment (orviwan) [inline README.md:1]: May aswell add a screenshot of the Pebble and perhaps a screenshot of the Node server output.

File: src/js/pebble-js-app.js (hunk 4 above)
Comment (C-D-Lewis) [inline src/js/pebble-js-app.js]: That's a good idea. One less thing that can go wrong.

**Comment by orviwan** [comment]
Can we talk about the data when you have a minute.

**Comment by orviwan** [comment]
👍

**Pull request merged**